doc/install.rst

```diff
@@ -0,0 +1,350 @@
+Installation
+############
+
+.. _looking_glass_client:
+
+Looking Glass Client
+--------------------
+
+This guide will step you through building the looking glass client from
+source, before you attempt to do this you should have a basic
+understanding of how to use the shell.
+
+.. _building_the_application:
+
+Building the Application
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _installing_build_dependencies:
+
+Installing Build Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These required libraries and tools should be installed first.
+
+.. _required_dependencies:
+
+Required Dependencies
+'''''''''''''''''''''
+
+-  cmake
+-  gcc \| clang
+-  fonts-freefont-ttf
+-  libegl-dev
+-  libgl-dev
+-  libfontconfig1-dev
+-  libgmp-dev
+-  libsdl2-dev
+-  libsdl2-ttf-dev
+-  libspice-protocol-dev
+-  make
+-  nettle-dev
+-  pkg-config
+
+.. _may_be_disabled:
+
+May be disabled
+<<<<<<<<<<<<<<<
+
+These dependencies are required by default, but may be omitted if their
+feature is disabled when running :ref:`cmake <client_building>`.
+
+-  Disable with ``cmake -DENABLE_BACKTRACE=no``
+
+   -  binutils-dev
+
+-  Disable with ``cmake -DENABLE_X11=no``
+
+   -  libx11-dev
+   -  libxfixes-dev
+   -  libxi-dev
+   -  libxss-dev
+
+-  Disable with ``cmake -DENABLE_WAYLAND=no``
+
+   -  libwayland-bin
+   -  libwayland-dev
+   -  wayland-protocols
+
+You can fetch these dependencies on Debian systems with the following command:
+
+``apt-get install binutils-dev cmake fonts-freefont-ttf libfontconfig1-dev
+libsdl2-dev libsdl2-ttf-dev libspice-protocol-dev libx11-dev nettle-dev
+wayland-protocols``
```

Comment:
Would this look better?

```suggestion
.. code:: bash

    apt-get install binutils-dev cmake fonts-freefont-ttf libfontconfig1-dev libsdl2-dev libsdl2-ttf-dev libspice-protocol-dev libx11-dev nettle-dev wayland-protocols
```

Reply:
pandoc did this by default, but it resulted in one line and a horizontal scroll bar, eww